camp/nfutsum/TOF_sensor/src/main/java/frc/robot/subsystems/ProximitySubsystem.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.playingwithfusion.TimeOfFlight;
+import com.playingwithfusion.TimeOfFlight.RangingMode;
+
+import edu.wpi.first.math.util.Units;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+
+public class ProximitySubsystem extends SubsystemBase {
+
+  // Defines Time of Flight sensor 
+  public static TimeOfFlight tofSensor = new TimeOfFlight(Constants.canID);
+
+  //Sets the ranging mode to medium (Between 76cm-290cm depending on lighting).
+  public void ProximitySubysytem(){
+    tofSensor.setRangingMode(RangingMode.Medium, 140);
+  }
+  
+
+public double getRange(){
+  //gets range and converts from millimeters to inches
```

Comment:
Please put a comment in the code that describes the emperical testing that you found using the TOF sensor.  I'd like to see a comment like:
// Found that TOF sensor minimum detection distance is X inches
// Found that TOF sensor maximum detection distance is Y inches
// Found that TOF sensor works best with a target made from MMMMM and is of size N by O inches in size.